platform/polkadot/api.go

```diff
@@ -125,14 +125,21 @@ func (p *Platform) NormalizeExtrinsic(srcTx *Extrinsic) *blockatlas.Tx {
 		Sequence: srcTx.Nonce,
 	}
 
+	value := "0"
+	if len(datas) > 0 {
```

Comment:
would it make sense to have similar function as: https://github.com/trustwallet/blockatlas/pull/777/files#diff-f77bff57c22cf320c2a91afaaef92572R22

Reply:
the logic's a little different